gnosis/safe-react
# Add address verification feedback

## What it solves
Resolves #2919

## How this PR fixes it
The ENS domain verification is seemingly working in `dev`. However, @liliya-soroka mentioned domain resolution taking too long.

Visual feedback is now given when a valid ENS/crypto domain is being resolved via a loading indicator.

## How to test it
1. Start the Safe loading process.
2. Enter a valid ENS domain.

- The field should be read only and a spinner shown in as an end adornment.
- If you type more characters, i.e. aaron.eth -> aaron.ethhhh, before the resolution is finished, the loading indicator should disappear and resolved address not be entered in field.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/139681588-b199b7de-30a9-4ab8-b9ec-73f9f82cde67.png)
